backend/src/config.ts

```diff
@@ -34,6 +34,96 @@ const config = convict({
     format: 'int',
     default: 8080,
   },
+  awsRegion: {
+    doc: 'The region in which the AWS service is running in',
+    env: 'AWS_REGION',
+    format: '*',
+    default: '',
+  },
+  nodeEnv: {
+    doc: 'The environment that NodeJS is running',
+    env: 'NODE_ENV',
+    format: '*',
+    default: 'development',
+  },
+  cookieMaxAge: {
+    doc: 'The maximum age for a cookie, expressed in ms',
+    env: 'COOKIE_MAX_AGE',
+    format: 'int',
+    default: 86400000, // 24 hours
+  },
+  dbHost: {
+    doc: 'The database host address',
+    env: 'DATABASE_HOST',
+    format: '*',
+    default: '',
+  },
+  dbName: {
+    doc: 'The database name',
+    env: 'DATABASE_NAME',
+    format: '*',
+    default: '',
+  },
+  dbPort: {
+    doc: 'The database port number',
+    env: 'DATABASE_PORT',
+    format: 'int',
+    default: '',
+  },
+  dbPassword: {
+    doc: 'The database password',
+    env: 'DATABASE_PASSWORD',
+    format: '*',
+    default: '',
+  },
+  dbUsername: {
+    doc: 'The database username',
+    env: 'DATABASE_USERNAME',
+    format: '*',
+    default: '',
+  },
+  dbDialect: {
+    doc: 'The database dialect',
+    env: 'DATABASE_DIALECT',
+    format: '*',
+    default: 'postgres',
+  },
```

Comment:
I think we can nest this config so that it becomes...
```suggestion
  db:{
    host: {
      doc: 'The database host address',
      env: 'DATABASE_HOST',
      format: '*',
      default: '',
    },
    database: {
      doc: 'The database name',
      env: 'DATABASE_NAME',
      format: '*',
      default: '',
    },
    port: {
      doc: 'The database port number',
      env: 'DATABASE_PORT',
      format: 'int',
      default: '',
    },
    password: {
      doc: 'The database password',
      env: 'DATABASE_PASSWORD',
      format: '*',
      default: '',
    },
    username: {
      doc: 'The database username',
      env: 'DATABASE_USERNAME',
      format: '*',
      default: '',
    },
    dialect: {
      doc: 'The database dialect',
      env: 'DATABASE_DIALECT',
      format: '*',
      default: 'postgres',
    },
  },
```